rpm/archivematica/package.spec

```diff
@@ -0,0 +1,340 @@
+/usr/lib/python2.7/archivematica/MCPClient/bin/pip install gearman
+/usr/lib/python2.7/archivematica/MCPClient/bin/pip install MySQL
+/usr/lib/python2.7/archivematica/MCPClient/bin/pip install django-extensions
+
+virtualenv --relocatable /usr/lib/python2.7/archivematica/MCPClient/
+cp -rf /usr/lib/python2.7/archivematica/MCPClient/* %{buildroot}/usr/lib/python2.7/archivematica/MCPClient/
+cp  %{_sourcedir}/%{name}/src/MCPClient/etc/archivematicaClientModules %{buildroot}/etc/archivematica/MCPClient/
+cp  %{_sourcedir}/%{name}/src/MCPClient/etc/clientConfig.conf %{buildroot}/etc/archivematica/MCPClient/
+cp -rf  %{_sourcedir}/%{name}/src/MCPClient/lib/* %{buildroot}/usr/lib/archivematica/MCPClient
+
+
+mkdir -p %{buildroot}/usr/lib/systemd/system/
+cat << EOF > %{buildroot}/usr/lib/systemd/system/archivematica-mcp-client.service
+[Unit]
+Description=Archivematica MCP Client
+After=syslog.target network.target
+
+[Service]
+User=archivematica
+EnvironmentFile=-/etc/sysconfig/archivematica-mcp-client
+ExecStart=/usr/lib/python2.7/archivematica/MCPClient/bin/python /usr/lib/archivematica/MCPClient/archivematicaClient.py
+
+[Install]
+WantedBy=multi-user.target
+
+EOF
+
+cat << EOF >  %{buildroot}/etc/sysconfig/archivematica-mcp-client
+DJANGO_SECRET_KEY=CHANGE_ME_WITH_A_SECRET_KEY
+ARCHIVEMATICA_MCPCLIENT_DB_NAME=MCP
+ARCHIVEMATICA_MCPCLIENT_DB_USER=archivematica
+ARCHIVEMATICA_MCPCLIENT_DB_PASSWORD=demo
+ARCHIVEMATICA_MCPCLIENT_DB_HOST=localhost
+ARCHIVEMATICA_MCPCLIENT_DB_PORT=3306
+EOF
+
+
+# Common
+mkdir -p %{buildroot}/usr/share/archivematica-common/
+cp  %{_sourcedir}/%{name}/src/archivematicaCommon/requirements/base.txt  %{buildroot}/usr/share/archivematica-common/requirements.txt
+
+mkdir -p %{buildroot}/usr/lib/archivematica/archivematicaCommon/
+cp -rf  %{_sourcedir}/%{name}/src/archivematicaCommon/lib/* %{buildroot}/usr/lib/archivematica/archivematicaCommon/
+
+mkdir -p %{buildroot}/etc/archivematica/archivematicaCommon/
+cp -rf  %{_sourcedir}/%{name}/src/archivematicaCommon/etc/* %{buildroot}/etc/archivematica/archivematicaCommon/
+
+mkdir -p %{buildroot}/var/archivematica/sharedDirectory
+cp -rf %{_sourcedir}/%{name}/src/MCPServer/share/sharedDirectoryStructure/* %{buildroot}/var/archivematica/sharedDirectory/
+
+
+%prep
+rm -rf /usr/share/archivematica
+rm -rf /usr/lib/python2.7/archivematica
+rm -rf %{_sourcedir}/*
+rm -rf %{buildroot}/*
+mkdir -p %{buildroot}/%{install_dir}
+
+git clone --branch stable/1.4.x --depth 1 --single-branch https://github.com/artefactual/archivematica %{_sourcedir}/%{name}
+
+cd %{_sourcedir}/%{name} && git submodule init && git submodule update
+
+%clean
+rm -rf %{buildroot}
+
+
+# Post install scripts
+%post
+# Dashboard
+echo "Creating archivematica user"
+userID=`id -u archivematica`
+
+if [ "${userID}" = 333 ]; then
+  echo "User archivematica exists"
+else
+  useradd --uid 333 --user-group --home /var/lib/archivematica/ archivematica
+fi
+
+
+echo "Creating archivematicadashboard user"
```

Comment:
IIRC `archivematicadashboard` is not being used in recent branches. Use `archivematica` instead.